test/suites/scheduling/suite_test.go

```diff
@@ -494,7 +494,7 @@ var _ = Describe("Scheduling", Ordered, ContinueOnFailure, func() {
 				},
 			}),
 		)
-		It("should provision a right-sized node when a pod has InitContainers (mixed resources)", func() {
+		It("should provision a right-sized node when a pod has InitContainers (mixed resources)", Label("flaky"), func() {
```

Comment:
I added a flakey label here but the last 10 have passed, so maybe its not needed?